sklearn/base.py

```diff
@@ -321,17 +319,23 @@ def _get_tags(self):
         else:
             overridden_tags = {}
 
-        # Collect tags of the direct parents.
-        # __bases__ gives the direct parents, not the whole MRO
-        for direct_parent_class in self.__class__.__bases__:
-            if (hasattr(direct_parent_class, '_more_tags')
-                    and direct_parent_class != self.__class__):
-                more_tags = direct_parent_class._more_tags(self)
-                more_tags = {key: val for (key, val) in more_tags.items()
-                             if key not in overridden_tags.keys()}
-
-                collected_tags = _update_if_consistent(collected_tags,
-                                                       more_tags)
+        collected_tags = {}
+        def _resolve_tags(c):
+            nonlocal collected_tags
+            if c is BaseEstimator or c is object:
+                return
+
+            for direct_parent_class in c.__bases__:
+                _resolve_tags(direct_parent_class)
```

Comment:
can't you do ``get_tags`` maybe?

Reply:
yeah I suppose there are other ways to wrap it up